internal/cmd/common/flags_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/hashicorp/boundary/internal/cmd/base"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
 // TestPopulateAttrFlags tests common patterns we'll actually be using. Note
```

Comment:
can you add some new tests in this file to cover the case where just a key is set?

Reply:
Yep

Reply:
Done!